src/main/java/duke/CommandHandler.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class CommandHandler {
+    public static void handleCommand(String userInput, TaskList tasks) throws DukeException {
+        String[] parsedCommand = Parser.parse(userInput);
+        String commandType = parsedCommand[0].toLowerCase();
+
+        switch (commandType) {
```

Comment:
i liked the naming of your different handle functions, they were very clear and easy to understand